tests/nominal/test_NominalToIntegerTransformer.py

```diff
@@ -118,55 +72,6 @@ def expected_df_1():
 
         return df
 
-    def test_check_is_fitted_called(self, mocker):
-        """Test that BaseTransformer check_is_fitted called."""
-        df = d.create_df_1()
-
-        x = NominalToIntegerTransformer(columns=["a", "b"])
-
-        x.fit(df)
-
-        expected_call_args = {0: {"args": (["mappings"],), "kwargs": {}}}
-
-        with ta.functions.assert_function_call(
-            mocker,
-            tubular.base.BaseTransformer,
-            "check_is_fitted",
-            expected_call_args,
-        ):
-            x.transform(df)
-
-    def test_not_dataframe_error_raised(self):
-        df = d.create_df_1()
-
-        x = NominalToIntegerTransformer(columns=["a", "b"])
-        x.fit(df, df["a"])
-
-        with pytest.raises(
-            TypeError,
-            match=f"{x.classname()}: X should be a pd.DataFrame",
-        ):
-            x.transform(X=[1, 2, 3, 4, 5, 6])
-
-    def test_super_transform_called(self, mocker):
-        """Test that BaseNominalTransformer.transform called."""
-        df = d.create_df_1()
-
-        x = NominalToIntegerTransformer(columns="a")
-
-        x.fit(df)
-
-        expected_call_args = {0: {"args": (x, d.create_df_1()), "kwargs": {}}}
-
-        with ta.functions.assert_function_call(
-            mocker,
-            tubular.base.BaseTransformer,
-            "transform",
-            expected_call_args,
-            return_value=d.create_df_1(),
-        ):
-            x.transform(df)
-
     def test_learnt_values_not_modified(self):
```

Comment:
this feels like a test that could be moved into a base class if we added a new fixture storing 'learnt_attributes' by transformer, thoughts on that? might create a ticket

Reply:
That's a good idea, please do.

Reply:
https://github.com/lvgig/tubular/issues/265